lib/pages/saved_places/saved_places.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:latlong/latlong.dart';
+
+import 'package:trufi_core/blocs/search_locations/search_locations_cubit.dart';
+import 'package:trufi_core/l10n/trufi_localization.dart';
+import 'package:trufi_core/pages/saved_places/location_tiler.dart';
+import 'package:trufi_core/widgets/dialog_edit_text.dart';
+
+import '../../trufi_models.dart';
+import '../../widgets/trufi_drawer.dart';
+import '../choose_location.dart';
+
+class SavedPlacesPage extends StatelessWidget {
+  static const String route = '/places';
+
+  const SavedPlacesPage({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final searchLocationsCubit = context.read<SearchLocationsCubit>();
+    final localization = TrufiLocalization.of(context);
+    final theme = Theme.of(context);
+    return Scaffold(
+      appBar: AppBar(title: Text(localization.menuYourPlaces)),
+      body: DefaultTabController(
+        length: 2,
+        child: Column(
+          children: [
+            const SizedBox(height: 10),
+            TabBar(
+              tabs: [
+                Container(
+                  padding: const EdgeInsets.all(10),
+                  child: Text(
+                    // TODO translate
+                    "Saved places",
+                    style: theme.textTheme.bodyText1,
+                  ),
+                ),
+                Container(
+                  padding: const EdgeInsets.all(10),
+                  child: Text(
+                    // TODO translate
+                    "Favorite places",
+                    style: theme.textTheme.bodyText1,
+                    maxLines: 2,
+                  ),
+                )
+              ],
+            ),
+            const SizedBox(height: 5),
+            Expanded(
+              child: TabBarView(
+                children: [
+                  Scaffold(
+                    body: BlocBuilder<SearchLocationsCubit, SearchLocationsState>(
+                      builder: (context, state) {
+                        return ListView(
+                          padding: const EdgeInsets.symmetric(horizontal: 16.0),
+                          children: [
+                            const SizedBox(height: 10),
+                            Builder(builder: (_) {
```

Comment:
Why do we need a Builder here if we do not use the context that it provides?

Reply:
Sorry, it's my mistake, I'll pay more attention to these kinds of mistakes